.github/workflows/release.yml

```diff
@@ -201,8 +201,6 @@ jobs:
       - uses: actions/checkout@v3
       - name: Install Rust
         uses: dtolnay/rust-toolchain@stable
-        with:
-          targets: ${{ matrix.target }}
```

Comment:
upload-rust-binary-action internally calls `rustup target add`, so this is unneeded.

https://github.com/taiki-e/upload-rust-binary-action/blob/a9a2c3d8f238f311a925a074bd6ef21f00115960/main.sh#L126-L129